x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_stat.tsx

```diff
@@ -6,58 +6,96 @@
 
 import React, { FC } from 'react';
 import { i18n } from '@kbn/i18n';
-import { EuiStat, EuiIconTip, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiStat, EuiIconTip, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import { REGRESSION_STATS } from '../../../../common/analytics';
 
 interface Props {
   isLoading: boolean;
   title: number | string;
-  isMSE: boolean;
+  statType: REGRESSION_STATS;
   dataTestSubj: string;
 }
 
-const meanSquaredErrorText = i18n.translate(
-  'xpack.ml.dataframe.analytics.regressionExploration.meanSquaredErrorText',
-  {
-    defaultMessage: 'Mean squared error',
-  }
-);
-const rSquaredText = i18n.translate(
-  'xpack.ml.dataframe.analytics.regressionExploration.rSquaredText',
-  {
-    defaultMessage: 'R squared',
-  }
-);
-const meanSquaredErrorTooltipContent = i18n.translate(
-  'xpack.ml.dataframe.analytics.regressionExploration.meanSquaredErrorTooltipContent',
-  {
-    defaultMessage:
-      'Measures how well the regression analysis model is performing. Mean squared sum of the difference between true and predicted values.',
-  }
-);
-const rSquaredTooltipContent = i18n.translate(
-  'xpack.ml.dataframe.analytics.regressionExploration.rSquaredTooltipContent',
-  {
-    defaultMessage:
-      'Represents the goodness of fit. Measures how well the observed outcomes are replicated by the model.',
-  }
-);
+const statDescriptions = {
+  [REGRESSION_STATS.MSE]: i18n.translate(
+    'xpack.ml.dataframe.analytics.regressionExploration.meanSquaredErrorText',
+    {
+      defaultMessage: 'Mean squared error',
+    }
+  ),
+  [REGRESSION_STATS.MSLE]: i18n.translate(
+    'xpack.ml.dataframe.analytics.regressionExploration.msleText',
+    {
+      defaultMessage: 'Mean squared logarithmic error',
+    }
+  ),
+  [REGRESSION_STATS.R_SQUARED]: i18n.translate(
+    'xpack.ml.dataframe.analytics.regressionExploration.rSquaredText',
+    {
+      defaultMessage: 'R squared',
+    }
+  ),
+  [REGRESSION_STATS.HUBER]: (
+    <FormattedMessage
+      id="xpack.ml.dataframe.analytics.regressionExploration.huberText"
+      defaultMessage="{wikiLink}"
+      values={{
+        wikiLink: (
+          <EuiLink
+            href="https://en.wikipedia.org/wiki/Huber_loss#Pseudo-Huber_loss_function"
+            target="_blank"
+          >
+            Pseudo Huber loss function
```

Comment:
This needs internationalizing.

Reply:
Updated in ba0dede3f530a09fd5219b8642bd2c1ca91ae907